lib/styles/build/css/design_tokens.css

```diff
@@ -97,7 +97,7 @@
   --vds-shadow-focus: var(--vds-shadow-hover);
   --vds-shadow-pressed: 0 1px 4px 1px rgba(45, 50, 59, 0.3);
   --vds-shadow-hover-error: 0 1px 4px 1px rgba(81, 88, 100, 0.3);
-  --vds-shadow-focus-error: var(--vds-shadow-hoverError);
+  --vds-shadow-focus-error: var(--vds-shadow-hover-error);
```

Comment:
This entire file and its JS counterpart are actually generated from a build tool called [style dictionary](https://virtru.github.io/virtuoso-design-system/?path=/docs/basics-design-tokens--page). If we need to make changes, they should be in the token JSON file: 
https://github.com/virtru/virtuoso-design-system/blob/master/lib/styles/style_dict/tokens/shadow.json

I'm not clear on why the variable name changes here…

Reply:
May be faster to simply duplicate the focus error token with the hover error value:

```json
"focusError":  { "value": "0 1px 4px 1px rgba(81, 88, 100, 0.3)" }
```

Reply:
@pflynn-virtru see if you can update shadow.json and (hopefully) follow these instructions to generate the token files (CSS vars and JS exports): https://github.com/virtru/virtuoso-design-system/blob/master/README.md#design-tokens

Reply:
@pflynn-virtru lmk if you need any help doing the above. Excited to fix this bug you found (and test out our documentation for design tokens).

Reply:
@pflynn-virtru @jrschumacher sorry for the delay on this. I fixed the problem at the source [in this commit](https://github.com/virtru/virtuoso-design-system/pull/72/commits/3178bf87f155df56dea352e81a050a5c09312ff3).